src/framework/mpas_stream_manager.F

```diff
@@ -47,7 +47,9 @@ module mpas_stream_manager
               MPAS_get_stream_filename, &
               MPAS_build_stream_filename
 
-    public :: postread_reindex
+    public :: prewrite_reindex, &
+              postwrite_reindex, &
+              postread_reindex
```

Comment:
Should we make a comment here for `prewrite_reindex` and `postwrite_reindex` that says that these routines should be only be used when calling `MPAS_writeStream` and not `mpas_stream_mgr_write`? I think that would clarify to an unfamiliar user that these routines are for ad-hoc streams.

Reply:
That seems like a good idea. I've pushed a new commit that adds notes in several places.

Reply:
Those notes look great!